subhub/app.py

```diff
@@ -106,9 +107,24 @@ def server_stripe_card_error(e):
 
     @app.app.before_request
     def before_request():
+        from flask import request
```

Comment:
let's move this to the top of the file with the rest of the Flask imports

Reply:
Cool, will do!